_posts/2023-10-17-AugustSeptember.md

```diff
@@ -0,0 +1,158 @@
+
+The game anniversary and 8th anniversary of the official server (`ctf.rubenwardy.com:30001`) were celebrated from August
+16th to September 1st, with fireworks and party hats as cosmetics on the server.
+
+{% include figure_gallery.html items=page.gallery_ctf %}
+
+### Glitch
+[The winner of the 2022 Minetest GAME JAM](https://content.minetest.net/packages/Wuzzy/glitch/) has continued
+development since then. [As of August](https://forum.minetest.net/viewtopic.php?p=428292#p428292), Glitch (by Wuzzy) has
+[a weblate](https://translate.codeberg.org/projects/glitch/) where the community can contribute game translations. You
+can read about the development history of Glitch on [Wuzzy's
+website](https://wuzzy.codeberg.page/games/makingof_glitch/).
+
+## Mods News
+
+### HTTP Blocks
+You can send `POST` and `GET` requests directly from Minetest using the new [HTTPBlocks mod by
+56independent](https://content.minetest.net/packages/56independent/httpblock/)! This
+[Digilines](https://content.minetest.net/packages/Jeija/digilines/) device allows you to construct complex systems that
+can talk to web endpoints and request plaintext files, records, and any other raw web data.
+
+The mod, of course, requires HTTP trust to use Minetest's HTTP API. Future plans include HTML querying using selector
+blocks, blocks to convert JSON to Lua tables, and maybe websocket support in the distant future.
+
+{% include figure_gallery.html items=page.gallery_httpblocks %}
+
+### Unicode Text Rendering
+The [Unicode Text library by erlehmann](https://content.minetest.net/packages/erlehmann/unicode_text/) is built to
+render fixed-width and variable-width [GNU Unifont fonts](http://savannah.gnu.org/projects/unifont/) (`.hex`). UTF-8
+text input is converted to a table of pixels that can be written to an image file (such as a
+[PNG](https://github.com/minetest/minetest/blob/5.7.0/doc/lua_api.txt#L5089-L5102) or
+[TGA](https://content.minetest.net/packages/erlehmann/tga_encoder/)). Non-Unicode text can also be written using the
+Unifont CSUR font.
+
+By rendering text to a single image server-side, the full range of Unicode can be supported by servers without sending
+thousands or tens of thousands of textures to Minetest clients. The [Unicode Signs mod by
+cora](https://content.minetest.net/packages/cora/ucsigns/) provides signs that use the Unicode Text library to draw
+text.
+
+{% include figure_gallery.html items=page.gallery_unicodetext %}
+
+## Art and Builds
+
+{% include figure.html src="/static/blog/2023_AugustSeptember/build_camper.png" caption='"Who wants to cook?" by
+1.1.8.4' %}
+
+{% include figure.html src="/static/blog/2023_AugustSeptember/build_cityhall.png" caption='"City hall, German Creative
+Server" by Toadie' %}
+
+{% include figure.html src="/static/blog/2023_AugustSeptember/build_computer.png" caption='"Mesecon-only (without
+Luacontrollers) computer" by 1F616EMO' %}
+
+## Server News
+
+### Archtec
+<sub>archtec.niklp.net:30803</sub>
```

Comment:
FYI you could link to `https://archtec.niklp.net/`, there are all necessary information.